sdks/go/pkg/beam/core/runtime/pipelinex/clone.go

```diff
@@ -64,10 +64,8 @@ func ShallowCloneParDoPayload(p *pb.ParDoPayload) *pb.ParDoPayload {
 
 	ret := &pb.ParDoPayload{
 		DoFn:               p.DoFn,
-		Splittable:         p.Splittable,
 		RestrictionCoderId: p.RestrictionCoderId,
```

Comment:
@robertwb 
It looks like we'll need to add the splittable dofn capability to the Go SDK.